udata/translations/fr/LC_MESSAGES/udata.po

```diff
@@ -245,7 +245,23 @@ msgstr "Formulaire de contact web de l'organisation"
 msgid "Publish as"
 msgstr "Publier en tant que"
 
-#: udata/core/contact_point/models.py:18 udata/tests/api/test_contact_points.py:50
+#: udata/core/contact_point/forms.py:21 udata/core/organization/forms.py:94
+msgid "Role"
+msgstr "Rôle"
+
+#: udata/core/contact_point/models.py:9
+msgid "Contact"
+msgstr "Contact"
+
+#: udata/core/contact_point/models.py:10
+msgid "Creator"
+msgstr "Créateur"
+
+#: udata/core/contact_point/models.py:11
+msgid "Publisher"
+msgstr "Éditeur"
```

Comment:
We've actually used `Publisher` as contact role but its translation is Éditeur in French.
Do we wanna modify it to `Producer / Producteur`? cc @agarrone

Reply:
We keep close to the DCAT vocabulary :)